src/Datadog.Trace/AppSec/Transports/Http/HttpTransport.Framework.cs

```diff
@@ -43,14 +43,14 @@ public IAdditiveContext GetAdditiveContext()
             return context.Items[WafKey] as IAdditiveContext;
         }
 
-        public Request Request()
+        public Request Request() => new()
         {
-            return new Request
-            {
-                Url = context.Request.Url,
-                Method = context.Request.HttpMethod,
-            };
-        }
+            Url = context.Request.Url,
+            Method = context.Request.HttpMethod,
+            Scheme = context.Request.Url.Scheme,
+            RemoteIp = context.Request.ServerVariables["HTTP_X_FORWARDED_FOR"] ?? context.Request.ServerVariables["REMOTE_ADDR"],
```

Comment:
How did we decide what's the right value, between `request.RemoteIp`, `context.Request.ServerVariables["HTTP_X_FORWARDED_FOR"]`, and `context.Request.ServerVariables["REMOTE_ADDR"]`? I think what you have is okay for now, but I think there's going to be an official algorithm to follow about how we should collect the IP address, so we may have to change this at somepoint.

Reply:
yes! let's confirm with the team how we get the remote ip later